src/VfacTmdb/Items/TVItemChanges.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types=1);
+/**
+ * This file is part of the Tmdb package.
+ *
+ * (c) Vincent Faliès <vincent@vfac.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Vincent Faliès <vincent@vfac.fr>
+ * @copyright Copyright (c) 2017-2021
+ */
+
+
+namespace VfacTmdb\Items;
+
+use VfacTmdb\Abstracts\Items\ItemChanges;
+use VfacTmdb\Exceptions\TmdbException;
+use VfacTmdb\Interfaces\TmdbInterface;
+
+/**
+ * TV Item Changes class
+ * @package Tmdb
+ * @author Steve Richter <steve@nerdbra.in>
+ * @copyright Copyright (c) 2021
+ */
+class TVItemChanges extends ItemChanges
+{
+    /**
+     * Constructor
+     * @param TmdbInterface $tmdb
+     * @param int $movie_id
+     * @param array $options
+     */
+    public function __construct(TmdbInterface $tmdb, int $movie_id, array $options = array())
+    {
+        try {
+            parent::__construct($tmdb, 'tv', $movie_id, $options);
+        } catch (TmdbException $ex) {
+            throw $ex;
+        }
+    }
+
+    /**
+     * @return \AppendIterator|Results\ItemChange
+     */
+    public function getSeasonChanges()
+    {
+        $seasonChanges = new \AppendIterator();
+        foreach ($this->getChangesByKey('season') as $change) {
+            if (null !== $change->getValueByKey('season_id')) {
+                $seasonChanges->append(
+                    (new TVSeasonItemChanges($this->tmdb, $change->getValueByKey('season_id'), $this->params))
+                        ->getChanges()
+                );
+            }
+        }
+
+        return $seasonChanges;
+    }
+
+    /**
+     * @return \AppendIterator|Results\ItemChange
+     */
+    public function getEpisodeChanges()
```

Comment:
I feel that both this and `getSeasonChanges()` are good methods to offer due to the way the API handles TV changes. Each change for a season or episode are listed as a change on the main TV item itself, but to find the specific changes you need to make additional requests with the returned ids until you get to the season/episode level. These helper functions will allow someone to access all of the episode changes for a TV show without having to handle the additional requests themselves. The methods may be better suited on the TVShow Item class.